### ISSUE TITLE
pdftex (started via the compiler driver) can't find input files if the main file is on a network drive

### ISSUE THREAD

**Issue by u-fischer** [MiKTeX/miktex#50]
I couldn't test the issue myself as I don't have access to a system with a network drive. I'm also not sure if it is a miktex or a texworks problem as I couldn't find someone with a texlive system to test. 

I had two independant reports that input files could not be found in the following setup:

* the main file was in a folder on a network drive with address like `//xxx/zz/yyy/test/`
* the input file (a text file in one case, a figure in the other) was in the same folder
* the file was opened in texworks (with double click in one case, unknown in the other)
* the compilation was started with the button in texworks.

In both cases latex complained that the input file could not be found.

The log-file showed the full path at the begin:

~~~~
entering extended mode
**"//xxx/zz/yyy/test/test.tex"
~~~~

The file compiled without problems if everything was copied to a local folder. The log-file then showed only the file name `**./text.tex`. It also compiled without problems on the network drive if texstudio instead of texworks was used. 

It looks as if texworks doesn't correctly set the "current folder" for such files.

**Comment by edocevoli** [comment]
This is not a TeXworks bug. Running:

```
texify --pdf --clean \\xxx\zz\yyy\test\test.tex
```

also fails.

**Comment by edocevoli** [comment]
And this fails too:

```
pdflatex \\xxx\zz\yyy\test\test.tex
```

**Comment by edocevoli** [comment]
This will be fixed with [milestone 2.9.6800](https://github.com/MiKTeX/miktex/milestone/7).
